### PR TITLE
Add Project Fi to utilities

### PR DESCRIPTION
**Pull request by mridulg**
Resolving issue #2116

### CODE REVIEW

File: _data/utilities.yml [hunk 1]
```diff
@@ -142,6 +142,16 @@ websites:
       img: powerstream.png
       twitter: PowerStreamNews
       tfa: No
+
```
Comment (stephengroat) [inline _data/utilities.yml]: Please cleanup the whitespace

**Review by stephengroat — CHANGES_REQUESTED** [review]
Whitespace cleanup

File: _data/utilities.yml (hunk 1 above)
Comment (mridulg) [reply _data/utilities.yml]: on it

Comment (mridulg) [reply _data/utilities.yml]: Done @stephengroat.

**Review by stephengroat — APPROVED** [review]

**Pull request merged**